src/pageText.js

```diff
@@ -170,54 +169,57 @@ async function addListener(listener, {
                 }
             }
         },
-        {
-            type: "webScience.pageText.parsedText",
-            schema: {
-                pageId: "string",
-                url: "string",
-                title: "string",
-                content: "string",
-                textContent: "string",
-                privateWindow: "boolean"
-            }
-        });
+            {
```

Comment:
did you mean to add this indentation here?

Reply:
Sorry this is my editor doing it, but I wonder if eslint isn't configured quite right in the repo.